### ISSUE TITLE
TransD model train failed when e_dim is different from r_dim

### ISSUE THREAD

**Issue by yyysfff** [thunlp/OpenKE#282]
Hello, I`m in trouble. While I`m going to run train_transd_FB15K237 with changing the r_dim from 200 to 300 and keeping the e_dim 200, it shows that func self._resize() and func.transfer() run error. My Pytorch version is 1.2.0.

**Comment by yyysfff** [comment]
我把_transfer中的e = F.normalize(
				self._resize(e, -1, r_transfer.size()[-1]) + torch.sum(e * e_transfer, -1, True) * r_transfer ,
				p = 2, 
				dim = -1
			)	
分别修改成了
			e = F.normalize(
				self._resize(e, 2, r_transfer.size()[-1]) + r_transfer * torch.sum(e * e_transfer, -1, True),
				p = 2, 
				dim = -1
			)		
和
return F.normalize(
				self._resize(e, 1, r_transfer.size()[-1]) + r_transfer * torch.sum(e * e_transfer, -1, True),
				p = 2, 
				dim = -1
			)
程序运行成功
